playbooks/adhoc/uninstall_openshift.yml

```diff
@@ -210,6 +210,11 @@
     failed_when: False
     with_items: "{{ exited_containers_to_delete.results }}"
 
+  - name: Delete all cri-o containers
+    shell: "crictl ps --all | cut -d ' ' -f 1 | grep -v CONTAINER | xargs crictl rm"
```

Comment:
```suggestion
    shell: "crictl ps --all -q | xargs crictl rm -f"
```

Reply:
@vrutkovs Thanks for the -q, the latest revision takes advantage of that. ptal